File: BackEnd/Api/Application/DataTransfer/StaffDto.cs
using Api.Domain.ValueObjects;

namespace Api.Application.DataTransfer;

public class StaffDto
{
	public string MechanographicNumber { get; set; }
	public string Name { get; set; }
	public string Email { get; set; }
	public string PhoneNumber { get; set; }
	public int Status { get; set; }
	public OperationalWindow OperationalWindow { get; set; }
	public ICollection<QualificationDto> Qualifications { get; set; }

}

public class CreateStaffDto
{
	public string MechanographicNumber { get; set; }
	public string Name { get; set; }
	public string Email { get; set; }
	public string PhoneNumber { get; set; }
	public int Status { get; set; }
	public OperationalWindow OperationalWindow { get; set; }
	public ICollection<string> QualificationsCodes { get; set; }

}

File: FrontEnd/src/model/dto/StaffDto.ts
import type { OperationalWindow } from "../values/OperationalWindow";

export interface StaffDto {
    mechanographicNumber: string;
    name: string;
    email: string;
    phoneNumber: string;
    status: number;
    operationalWindow: OperationalWindow;
    qualificationsCodes?: string[];
}